src/components/Comment.jsx

```diff
@@ -3,10 +3,13 @@ import Voter from "./Voter";
 
 
 export default function Comment({ data, }) {
+  const voteRequestConf = {
+    url: `/articles/${data.article_id}/comments`
+  };
```

Comment:
This doesn't look to be the right url for voting on a comment - wouldn't we be using the comment_id instead? This component currently doesn't work / isn't completed - remember that we want to keep our PRs as a relatively clean and polished version of the code, so we should try avoid WIP stuff - this is because code on a PR is making it's way towards the main version of the app

And since the task is voting on an article, this seems to be going over what's needed in this PR